core/CMakeLists.txt

```diff
@@ -682,11 +684,13 @@ else (UNIX)
   else (X64)
     set(LD_FLAGS "${LD_FLAGS} /machine:I386")
   endif (X64)
+
```

Comment:
Unintended change?

Reply:
I don't see a reply: generally these would be reverted

Reply:
Done